Didn't find class "kotlin.jvm.internal.Lambda" on path: DexPathList...

When you follow [Mobly-101 tutorial](https://github.com/google/mobly/blob/master/docs/tutorial.md) and run the following command:
`python3 hello_world_test.py -c sample_config.yml`
You see following error:
```
Process crashed before executing the test(s):
java.lang.NoClassDefFoundError: Failed resolution of: Lkotlin/jvm/internal/Lambda;
	at androidx.test.platform.io.FileTestStorage.<init>(FileTestStorage.java:45)
	at androidx.test.runner.AndroidJUnitRunner.registerTestStorage(AndroidJUnitRunner.java:664)
	at androidx.test.runner.AndroidJUnitRunner.onStart(AndroidJUnitRunner.java:441)
	at com.google.android.mobly.snippet.SnippetRunner.onStart(SnippetRunner.java:139)
	at android.app.Instrumentation$InstrumentationThread.run(Instrumentation.java:2261)
Caused by: java.lang.ClassNotFoundException: Didn't find class "kotlin.jvm.internal.Lambda" on path: DexPathList[[zip file "/data/app/~~TqH6aDW44-ek2Rxsb9RFAQ==/com.google.android.mobly.snippet.bundled-x62zokKJRiTiyC0a9LL1xA==/base.apk"],nativeLibraryDirectories=[/data/app/~~TqH6aDW44-ek2Rxsb9RFAQ==/com.google.android.mobly.snippet.bundled-x62zokKJRiTiyC0a9LL1xA==/lib/arm64, /system/lib64, /system/system_ext/lib64]]
	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:259)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
	... 5 more
INSTRUMENTATION_RESULT: shortMsg=Process crashed.
```
If you following [this Mobly discussion](https://github.com/google/mobly-bundled-snippets/issues/173), it resolves by adding the following to `build.gradle`:
`implementation 'org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.6.10'`

---

## Regarding contribution guidelines
I didn't update any of these files, but running `./gradlew presubmit` gives me `Not all files were formatted`.
Checkout out the scan results [here](https://gradle.com/s/l6a74yltgu4sg).

Do let me know if you need anything else.

Thanks!

--- a/build.gradle
+++ b/build.gradle
@@ -4,7 +4,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.3.1'
+        classpath 'com.android.tools.build:gradle:7.0.4'
 
         // NOTE: Do not place your application dependencies here.
     }
@@ -76,6 +76,7 @@ dependencies {
     implementation 'com.google.code.gson:gson:2.8.6'
     implementation 'com.google.guava:guava:31.0.1-jre'
     implementation 'com.google.errorprone:error_prone_annotations:2.15.0'
+    implementation 'org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.6.10'
 
     testImplementation 'com.google.errorprone:error_prone_annotations:2.15.0'
     testImplementation 'com.google.guava:guava:31.0.1-jre'

--- a/build.gradle
+++ b/build.gradle
@@ -4,7 +4,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.4'
+        classpath 'com.android.tools.build:gradle:7.3.1'
 
         // NOTE: Do not place your application dependencies here.
     }

